composer/callbacks/run_directory_uploader.py

```diff
@@ -40,15 +40,28 @@ class RunDirectoryUploader(Callback):
     timestamp. Only files that have a newer last modified timestamp since the last upload will be  uploaded.
 
     Example
-        .. testsetup:: *
+        .. testsetup::
+
+           import os
+           import functools
+           from composer.callbacks import RunDirectoryUploader, run_directory_uploader
```

Comment:
This doctest was never working properly, but because it spawned subprocesses, it had a race condition that allowed the doctest build to pass. This fix is unrelated to this PR, but including it here since it was causing the doctest to fail occasionally when testing locally.